dojo/tools/blackduck/importer.py

```diff
@@ -103,12 +120,21 @@ def _process_project_findings(self, project_ids, security_issues, files=None):
                     security_issue_dict.get('Remediation target date'),
                     security_issue_dict.get('Remediation actual date'),
                     security_issue_dict.get('Remediation comment'),
-                    ', '.join(locations)
+                    location
                 )
 
-    def __partition_by_project_id(self, csv_file):
+    def __partition_by_key(self, csv_file):
         records = csv.DictReader(csv_file)
         findings = defaultdict(set)
+        # Backwards compatibility. Newer versions of Blackduck use Component id.
+        if "Project id" in records.fieldnames:
+            key = "Project id"
+        else:
+            key = "Component id"
         for record in records:
-            findings[record.get('Project id')].add(frozenset(record.items()))
+            findings[record.get(key)].add(frozenset(record.items()))
         return findings
+
+    def get_cve(self, vuln_id):
+        cve_search = re.search("CVE-\d{4}-\d{4,7}", vuln_id, re.IGNORECASE)
```

Comment:
I think this would need to be `re.search(r"CVE-\d{4}-\d{4,7}", vuln_id, re.IGNORECASE)` in order for it to be flake8 compliant .